### DISCUSSION TITLE
Products available on database and frontend but not via gql query

### DISCUSSION THREAD

**Discussion by dialformonkey** [saleor/saleor#8652] [Q&A]
Hi. ive recently started to encounter an issue with my linode instance where i cant query products created on the saleor dashboard. It is correctly being updated/deleted on my database however when i query for the items i am returned with a blank response.

Could this be an issue with my redis instance perhaps?

<img width="1235" alt="Screen Shot 2021-11-25 at 4 56 51 AM" src="https://user-images.githubusercontent.com/62189575/143327007-c19d6e92-943c-4d73-8589-cc7818ef02ea.png">

<img width="1298" alt="Screen Shot 2021-11-25 at 4 56 34 AM" src="https://user-images.githubusercontent.com/62189575/143326987-838996b5-0291-4e4c-9c00-1f0424f016a5.png">

<img width="234" alt="Screen Shot 2021-11-25 at 4 56 12 AM" src="https://user-images.githubusercontent.com/62189575/143326959-7885a612-6258-4bad-b7aa-d7e68cd4110b.png">
e

**Comment by karolkielecki** [comment]
@dialformonkey Hi! Could you please provide more info? Which version are you using?
